vaccine_feed_ingest/runners/tn/vaccinate_gov/normalize.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+
+# this dataset produces duplicates when a single location offers multiple
+# vaccine types, or offers both first and second doses. we make no effort to
+# deduplicate them here, though lat/long should be sufficient to do so
+# downstream.
+
+import datetime
+import json
+import logging
+import pathlib
+import sys
+from typing import List, Optional
+
+from vaccine_feed_ingest_schema import schema  # noqa: E402
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)s:%(name)s:%(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+)
+logger = logging.getLogger("tn/vaccinate_gov/normalize.py")
+
+SOURCE_NAME = "tn_vaccinate_gov"
+
+
+def _get_id(site: dict) -> str:
+    return site["id"].replace(" ", "-")
+
+
+def _get_name(site: dict) -> str:
+    # most names are of the form "Descriptive Name - Vaccine available"
+    # we will attempt to drop the latter token where it exists
+    sep = " - "
+    if sep in site["title"]:
+        return sep.join(site["title"].replace("\u2013", "-").split(sep)[:-1])
+    else:
+        return site["title"]
+
+
+def _get_vaccine_description(site: dict) -> str:
+    # most names are of the form "Descriptive Name - Vaccine available"
+    # we will attempt to return the last token (or the whole string)
+    sep = " - "
+    return site["title"].replace("\u2013", "-").split(sep)[-1]
```

Comment:
For the names that don't have the token, it looks like this will return the name. Should we return `None` in those cases instead?

Ultimately this shouldn't matter since this is currently used only for the regex.